python/mlcroissant/mlcroissant/_src/structure_graph/base_node.py

```diff
@@ -217,18 +218,22 @@ def from_jsonld(cls, *args, **kwargs) -> Any:
     def validate_name(self):
         """Validates the name."""
         name = self.name
+        if not name:
+            # This case is already checked for in every node's __post_init__ as `name`
+            # is a mandatory parameter for Croissant 0.8
+            return
```

Comment:
Still raise an error in case someone deletes the condition in `__post_init__`?

Reply:
Sure -- I just copy-pasted this part of the code from below, though. Let me quickly check that it won't interfere with previous 0.8 tests